tt_metal/hw/ckernels/grayskull/llk_lib/llk_unpack_AB.h

```diff
@@ -170,4 +135,8 @@ inline void llk_unpack_AB(
 
     // Switch unpacker config context
     switch_config_context(unp_cfg_context);
+
+#ifdef PERF_DUMP
+    first_unpack_recorded = true;
+#endif
```

Comment:
Will mention one more time, but this should apply everywhere. There should not be any perf dump code blocks, we don't support the flow used in budabackend.

Reply:
agree we will have to remove perf dump code, but problem is right now is that buda perf dump hooks are ingrained in the llk_lib kernels, and will cause all buda perf test sweeps to fail if removed now (because this file will be parity with buda). But I agree @acejkov and I will review and separate these perf hooks to be in the buda llk_api files once these files are commonalized and we can make sure buda regressions don't fail.

Reply:
there might be an option to reuse perf dump tools for metal. we keep as integral part of the LLK and higher level stack can decide to use it or not